girder/models/__init__.py

```diff
@@ -58,14 +69,17 @@ def getDbConnection(uri=None, replicaSet=None):
         uri = dbConf.get('uri')
         replicaSet = dbConf.get('replica_set')
     clientOptions = {
-        'connectTimeoutMS': 15000,
```

Comment:
The default for this is 20000 ms, and has been officially since pymongo 2.7.  We were using that version when I added this parameter, and at the time the default was not being honored.  Do we know for certain that the default is now being used so that it is safe to remove this?